src/OctoshiftCLI.Tests/Octoshift/Commands/AbortMigration/AbortMirationCommandHandlerTests.cs

```diff
@@ -0,0 +1,65 @@
+using System.Threading.Tasks;
+using Moq;
+using OctoshiftCLI.Commands.AbortMigration;
+using OctoshiftCLI.Services;
+using Xunit;
+
+namespace OctoshiftCLI.Tests.Octoshift.Commands.AbortMigration;
+
+public class AbortMigrationCommandHandlerTests
+{
+    private readonly Mock<GithubApi> _mockGithubApi = TestHelpers.CreateMock<GithubApi>();
+    private readonly Mock<OctoLogger> _mockOctoLogger = TestHelpers.CreateMock<OctoLogger>();
+
+    private readonly WarningsCountLogger _warningsCountLogger;
+    private readonly AbortMigrationCommandHandler _handler;
+
+    private const string REPO_MIGRATION_ID = "RM_MIGRATION_ID";
+
+    public AbortMigrationCommandHandlerTests()
+    {
+        _warningsCountLogger = new WarningsCountLogger(_mockOctoLogger.Object);
+        _handler = new AbortMigrationCommandHandler(_mockOctoLogger.Object, _mockGithubApi.Object);
+
+        TestHelpers.SetCliContext();
+    }
+
+    [Fact]
+    public async Task With_Migration_ID_That_Succeeds()
+    {
+        // Arrange
+        _mockGithubApi.Setup(x => x.AbortMigration(It.IsAny<string>())).ReturnsAsync(true);
+
+        // Act
+        var args = new AbortMigrationCommandArgs
+        {
+            MigrationId = REPO_MIGRATION_ID,
+        };
+        await _handler.Handle(args);
+
+        // Assert
+        _mockOctoLogger.Verify(m => m.LogSuccess(It.IsAny<string>()), Times.Once);
+        _mockGithubApi.Verify(m => m.AbortMigration(REPO_MIGRATION_ID), Times.Exactly(1));
+        _mockGithubApi.VerifyNoOtherCalls();
+    }
+
+    [Fact]
+    public async Task With_Migration_ID_That_Fails()
+    {
+        // Arrange
+        _mockGithubApi.Setup(x => x.AbortMigration(It.IsAny<string>())).ReturnsAsync(false);
+
+        // Act
+        var args = new AbortMigrationCommandArgs
+        {
+            MigrationId = REPO_MIGRATION_ID,
+        };
+        await _handler.Handle(args);
+
+        // Assert
+        _mockOctoLogger.Verify(m => m.LogSuccess(It.IsAny<string>()), Times.Once);
+        _mockGithubApi.Verify(m => m.AbortMigration(REPO_MIGRATION_ID), Times.Exactly(1));
+        _mockGithubApi.VerifyNoOtherCalls();
+    }
```

Comment:
I would remove this test since the only difference is the log output. a similar example is [RevokeMigrationRoleCommandHandlerTests.cs](https://github.com/github/gh-gei/blob/a6d92bb445ac5ad58117cfe6cedf94d3121fc569/src/OctoshiftCLI.Tests/Octoshift/Commands/RevokeMigratorRole/RevokeMigratorRoleCommandHandlerTests.cs) that only tests for a happy path. 
```suggestion
```